spec/dummy/lib/templates/haml/scaffold/index.html.haml

```diff
@@ -14,21 +14,21 @@
     - if show_actions
           %th.actions Actions
 
-      - @<%= plural_table_name %>.each do |<%= singular_table_name %>|
+      - @<%= plural_table_name %>.each do |record|
```

Comment:
renamed this to `record` so the boilerplate is more resilient when copying/pasting etc